### PR TITLE
Use GitHub actions to build pdf

### PR DESCRIPTION
**Pull request by bkmgit**
https://github.com/sfc-aqua/Overview-of-Quantum-Communications-E/issues/1

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        echo ${{ env.EVENT_NUMBER }}
+        echo ${{ env.URL }}
+        echo ${{ env.OWNER }}
+        echo ${{ env.REPO }}
+        echo ${{ env.RUN_URL }}
+    - name: Install TeXlive
+      run: sudo apt-get install texlive texlive-publishers texlive-science latexmk
+    - name: LaTeX Compile
+      run: pdflatex main.tex
+      run: pdflatex main.tex
```
Comment (zigen) [inline .github/workflows/build.yaml:32]: could you delete the duplicated line?

**Review by zigen — CHANGES_REQUESTED** [review]
thanks for contributing! I left a comment, could you fix it?

File: .github/workflows/build.yaml (hunk 1 above)
Comment (bkmgit) [reply .github/workflows/build.yaml:32]: Need to run it twice to ensure references and internal links are in final pdf.

Comment (rdviii) [reply .github/workflows/build.yaml:32]: Bkm is right.

Comment (zigen) [reply .github/workflows/build.yaml:32]: then, it should be 
```
- name: LaTeX Compile
  run: |
    pdflatex main.tex
    pdflatex main.tex
```

Comment (zigen) [reply .github/workflows/build.yaml:32]: > The workflow is not valid. .github/workflows/build.yaml (Line: 32, Col: 7): 'run' is already defined

https://github.com/sfc-aqua/Overview-of-Quantum-Communications-E/actions/runs/4540857355

**Comment by bkmgit** [comment]
Closing in favor of https://github.com/sfc-aqua/Overview-of-Quantum-Communications-E/pull/5

**Pull request closed without merging**